views/registration.ejs

```diff
@@ -0,0 +1,110 @@
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+        <!-- fontawesome cdn -->
+        <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+
+        <!-- css link-->
+        <link rel="stylesheet" href="css/login.css">
+
+
+    </head>
+    <body>
+          <div class="container">
+            <div class="row">
+              <div class="col-sm-9 col-md-7 col-lg-5 mx-auto">
+                <div class="fadeInDown">
+                <div class="card card-signin my-5">
+                  <div class="card-body">
+                      <div class="login100-form-title p-b-33">
+                          <div class="fadeIn first">
+                          <h3 class="login100-form-title" style="text-align: center">Blood Donation</h3>
+                          <img id="nitt-logo" src="img/nitt_logo.png">
+                          </div>
+                          <div class="fadeIn second">
+                          <h5 class="card-title text-center">Register</h5>
+                          </div>
+                      </div>
+                    <form class="form-signin" method="POST" action="/">
+                      <div class="fadeIn third">
+                      <div class="form-label-group">
+                          <input type="text" id="name" class="form-control" placeholder="Full Name" required autofocus>
+                          <label for="name">Name</label>
+                        </div>
+
+                      <div class="form-label-group">
+                        <input type="integer" id="mailid" class="form-control" placeholder="Email" required autofocus>
+                        <label for="mailid">Email</label>
+                      </div>
+
+                      <div class="form-label-group">
+                          <input type="date" id="Date" class="form-control" required>
+                          <label for="Date">Date of Birth</label>
+                      </div>
+
+                      <div class="form-label-group">
+                        <input type="password" id="Password" class="form-control" placeholder="Password" onkeyup='check();' required>
+                        <label for="Password">Password</label>
+                      </div>
+
+                      <div class="form-label-group">
+                        <input type="password" id="Confirm_Password" class="form-control" placeholder="Password" onkeyup='check();' required>
+                        <label for="Password">Confirm Password</label>
+                        <br>
+                        <span id="Message"></span>
+                      </div>
+
+                      <!-- To check the password and confirm-passowrd are the same- -->
+                      <script>
```

Comment:
INITIALLY: Have form submission feature disabled
The form should be submittable only after the password and confirm_password matches. 
This would be the last change required